src/pages/cache/purge-cache/PurgeCache.jsx

```diff
@@ -0,0 +1,144 @@
+import React, { useState, useEffect } from "react";
+import { useParams } from "react-router-dom";
+import { Form, Radio, Col, Card, Alert, Button, AutoComplete } from "antd";
+import { useSelector } from "react-redux";
+import Sidenav from '../../../components/sidenav/Sidenav'
+import Topbar from '../../../components/topbar/Topbar'
+import ProjectPageLayout, { Content, InnerTopBar } from "../../../components/project-page-layout/ProjectPageLayout"
+import { projectModules } from "../../../constants";
+import RadioCards from "../../../components/radio-cards/RadioCards"
+import FormItemLabel from "../../../components/form-item-label/FormItemLabel"
+import ConditionalFormBlock from "../../../components/conditional-form-block/ConditionalFormBlock";
+import { getDbConfigs, loadDbConfig, getTrackedCollections, loadDbRules, loadDbSchemas } from "../../../operations/database";
+import { notify, decrementPendingRequests, incrementPendingRequests } from "../../../utils";
+import { purgeCache } from "../../../operations/cache";
+
+const PurgeCache = () => {
+
+    const { projectID } = useParams();
+
+    useEffect(() => {
+        if (projectID) {
+            incrementPendingRequests()
+            loadDbConfig(projectID)
+                .catch(ex => notify("error", "Error fetching database config", ex))
+                .finally(() => decrementPendingRequests())
+
+            incrementPendingRequests()
+            loadDbSchemas(projectID)
+                .catch(ex => notify("error", "Error fetching database schemas", ex))
+                .finally(() => decrementPendingRequests())
+
+            incrementPendingRequests()
+            loadDbRules(projectID)
+                .catch(ex => notify("error", "Error fetching database rules", ex))
+                .finally(() => decrementPendingRequests())
+        }
+    }, [])
+
+    const [form] = Form.useForm();
+    const [selectedDb, setSelectedDb] = useState("");
+    const [value, setValue] = useState("");
+    const dbConfigs = useSelector(state => getDbConfigs(state))
+    const trackedCollections = useSelector(state => getTrackedCollections(state, selectedDb))
+
+    const dbList = Object.keys(dbConfigs)
+
+    const handleFinish = (values) => {
+        let data = {};
+        switch (values) {
```

Comment:
the switch should be on `values.level`. Confirm from the developer tools whether the correct request body is being posted or not for all the three types.